client/components/place/place-list/place-list.component.js

```diff
@@ -139,54 +147,127 @@ angular.module('placeList', ['filterMapType', 'popularTracks', 'ngAnimate'])
         ctrl.placesType = mapMarkingTypes.places;
         placeTypeLength = Object.keys(ctrl.placesType).length;
         placesOnMap.removePlaces();
-        placesOnMap.showMap();
         placesOnMap.initGroupsOfPlaces(ctrl.placesType);
+        map = placesOnMap.showMap();
```

Comment:
If so, remove `var map` and add `var` here.
Initializing variables immediately after declaration is a good practice.